src/vs/platform/dialogs/common/dialogs.ts

```diff
@@ -22,6 +22,33 @@ export interface ICheckbox {
 	checked?: boolean;
 }
 
+export interface IConfirmDialogArgs {
+	confirmation: IConfirmation;
+}
+
+export interface IShowDialogArgs {
```

Comment:
I suggest to introduce a common super-type of `IShowDialogArgs` and `IInputDialogArgs` to share properties

Reply:
made inputargs extend showargs following the pattern of result